src/addons/outputSpokenText.js

```diff
@@ -540,15 +557,13 @@ MathAtom.toSpeakableFragment = function(atom, options) {
 /**
  * @param {MathAtom[]}  [atoms] The atoms to represent as speakable text.
  * If omitted, `this` is used.
- * @param {Object.<string, any>} [options]
+ * @param {Object.<string, any>} [speechOptions]
 */
-MathAtom.toSpeakableText = function(atoms, options) {
-    if (!options) {
-        options = {
-            textToSpeechMarkup: '',     // no markup
-            textToSpeechRules: 'mathlive'
-        }
-    }
+MathAtom.toSpeakableText = function(atoms, speechOptions) {
+    const options = speechOptions ? JSON.parse(JSON.stringify(speechOptions)) : {
```

Comment:
Was this change upstream or our change? If our change, any particular reason, this is doing a .stringify and then immediately doing a .parse, since I thought these were opposite actions?

Reply:
@gavin-brown upstream 🤷‍♂